src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
+
+      return calculator.result;
+    },
+
+    subtract(number) {
+      calculator.result -= number;
+
+      return calculator.result;
+    },
+
+    multiply(number) {
+      calculator.result *= number;
+
+      return calculator.result;
+    },
+
+    divide(number) {
+      calculator.result /= number;
+
+      return calculator.result;
+    },
+
+    operate(callback, number) {
+      calculator.result = callback(number);
+
+      return this;
+    },
```

Comment:
Let's improve this part and use `this`

```suggestion
    operate(operation, value) {
      operation.call(this, value);

      return this;
    },
```

Reply:
воно не працює так нажаль, з менторами в чаті вирішували